test/image-blueprints-bootc/layer2-source/group2/rhel96-bootc-source-isolated.containerfile

```diff
@@ -17,30 +17,25 @@ RUN printf '\
 BLOCK_RPCS=\n\
 FSFREEZE_HOOK_PATHNAME=/etc/qemu-ga/fsfreeze-hook\n' > /etc/sysconfig/qemu-ga
 
-# SOURCE_IMAGES environment contains a comma-separated list of container image
-# references to be set by the calling procedure. The list is prepended one more
-# image used in the offline tests.
-# Split the variable and pull each image in a separate layer.
+# Embed images based on contents of release-info RPMs.
 #
 # Note:
-# - Gomplate blocks are commented out to avoid hadolint warnings.
 # - Retries work around sporadic "cannot set user namespace" podman error.
-#
-# {{ $SOURCE_IMAGES := (printf "%s,%s" "quay.io/microshift/busybox:1.36" (.Env.SOURCE_IMAGES)) }}
-# {{ range ($SOURCE_IMAGES  | strings.Split ",") }}
 RUN --mount=type=secret,id=pullsecret,dst=/run/secrets/pull-secret.json \
-    for i in 1 2 3; do \
-        GOMAXPROCS=8 podman pull \
-            --authfile /run/secrets/pull-secret.json \
-            --root /usr/lib/containers/storage \
-            "docker://{{ . }}" && break; \
-        if [ $i -eq 3 ] ; then \
-            echo "ERROR: Failed to pull {{ . }} image after 3 attempts"; \
-            exit 1; \
-        fi; \
-        echo "Attempt $i failed. Retrying in 5 seconds..." && sleep 5; \
+    images="$(jq -r '.images | .[]' /usr/share/microshift/release/release-"$(uname -m)".json)" ; \
```

Comment:
I think we're missing `quay.io/microshift/busybox:1.36` and we're getting offline failurers (speculation: pod cannot start because the image is missing)